modules/glean/runtime/glean-plugin.client.ts

```diff
@@ -33,24 +33,41 @@ export default defineNuxtPlugin((nuxtApp) => {
       }
     }
 
+    function recordEngagement(element: Element) {
+      if (!element)
```

Comment:
While the callers of this function also do checking for `element`, I still did it here to remove the optional chaining in the original - `eventTarget?.getAttribute()`